.github/workflows/lint.yml

```diff
@@ -9,22 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set local Go version
```

Comment:
🤔 Curious about the linting updates in this PR - what do you think about moving those to a separate PR?

Reply:
dropped the commit